src/database.rs

```diff
@@ -513,6 +516,31 @@ impl Database {
         self.verify.as_ref().unwrap()
     }
 
+    /// Get a statement by label.
+    pub fn statement(&mut self, name: &str) -> Option<StatementRef> {
+        match self.name_result().lookup_label(name.as_bytes()) {
+            None => None,
+            Some(lookup) => Some(self.parse_result().statement(lookup.address)),
+        }
+    }
+
+    /// Export an mmp file for a given statement.
+    pub fn export(&mut self, stmt: String) {
+        time(&self.options.clone(), "export", || {
+            let parse = self.parse_result().clone();
+            let scope = self.scope_result().clone();
+            let name = self.name_result().clone();
+            let sref = self.statement(&stmt)
+                .expect(format!("Label {} did not correspond to an existing statement",
```

Comment:
If you want to be really fancy, you could catch and reformat the `panic!` error message; obviously that's not recommended as a general try-catch mechanism, but for wrapping `main()` an sensibly displaying the error it's a fairly common style. (Similarly, in Java catching `Throwable` is not recommended, but it is often used at the top level to display a "fatal error" dialog instead of barfing on the console.)